melosys-eessi-app/src/main/java/no/nav/melosys/eessi/integration/eux/rina_api/EuxConsumer.java

```diff
@@ -314,21 +329,33 @@ public String getFilename() {
 
         return exchange("/rinasaker?buctype={buctype}&status={status}", HttpMethod.GET,
                 new HttpEntity<>(defaultHeaders()),
-                new ParameterizedTypeReference<List<BucInfo>>() {},
+                new ParameterizedTypeReference<List<BucInfo>>() {
+                },
                 bucType, status);
     }
 
+    public String hentRinaUrl(String rinaSaksnummer) {
```

Comment:
Test til denne?

Reply:
Utvidet testen som lå der fra før. Legger til en test til for sikkerhetsskyld

Reply:
Du har utvidet testen i `EuxService` ja. Men denne metoden er helt ny, og vi bør legge til en liten test for selve REST-kallet.
Du kan teste ut endepunktet fra https://eux-rina-api-q2.nais.preprod.local/swagger-ui.html#/eux-rina-api/lagUrlTilRinaSakSedUsingGET (fra utviklingsimage) for å se content-type som vi mottar.